docs/src/app/components/Navigation/docs.js

```diff
@@ -0,0 +1,391 @@
+import nextjs from './nextjs';
+'use strict';
+
+export default {
+  url: 'docs',
+  displayName: 'Docs',
+  children: [
+    {
+      url: 'getting-started',
+      displayName: 'Getting Started',
+      children: [
+        {
+          url: 'quick-start',
+          displayName: '1. Quick Start',
+        },
+        {
+          url: 'first-component',
+          displayName: '2. Your First Component',
+        },
+        {
+          url: 'jss-server-install',
+          displayName: '3. JSS Server Setup',
+        },
+        {
+          url: 'app-deployment',
+          displayName: '4. App Deployment',
+        },
+      ],
+    },
+    {
+      url: 'fundamentals',
+      displayName: 'Fundamentals',
+      children: [
+        {
+          url: 'architecture',
+          displayName: 'Architecture',
+        },
+        {
+          url: 'understanding-layout',
+          displayName: 'Understanding layout',
+        },
+        {
+          url: 'personalization',
+          displayName: 'Personalization',
+        },
+        {
+          url: 'dev-workflows',
+          displayName: 'Developer workflows',
+          children: [
+            {
+              url: 'overview',
+              displayName: 'Overview',
+            },
+            {
+              url: 'code-first',
+              displayName: 'Code-first',
+            },
+            {
+              url: 'sitecore-first',
+              displayName: 'Sitecore-first',
+            },
+          ],
+        },
+        {
+          url: 'application-modes',
+          displayName: 'Application modes',
+        },
+        {
+          url: 'cli',
+          displayName: 'JSS CLI',
+        },
+        {
+          url: 'services',
+          displayName: 'Services and APIs',
+          children: [
+            {
+              url: 'app-configuration',
+              displayName: 'App Configuration',
+            },
+            {
+              url: 'app-import',
+              displayName: 'App Import',
+            },
+            {
+              url: 'layout-service',
+              displayName: 'Layout Service',
+            },
+            {
+              url: 'graphql',
+              displayName: 'GraphQL',
+            },
+            {
+              url: 'tracking',
+              displayName: 'Tracking',
+            },
+            {
+              url: 'forms-service',
+              displayName: 'Forms',
+            },
+            {
+              url: 'dictionary-service',
+              displayName: 'Dictionary Service',
+            },
+            {
+              url: 'view-engine',
+              displayName: 'View Engine / SSR',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      url: 'client-frameworks',
+      displayName: 'Client Frameworks',
+      children: [
+        {
+          url: 'nextjs',
```

Comment:
This conflicts with `url: 'nextjs` in `docs/src/app/components/Navigation/nextjs.js` (both navs unfold when clicking any of the nextjs links). If it's something we plan on doing later (after merging client-frameworks/nextjs topics into the main nextjs docs) ignore me.

Reply:
@AncaIO  - you marked this as resolved, does this still need to be looked into?